source/documentation/common/upgrade/proc-Upgrading_hypervisor_preserve_gluster_storage.adoc

```diff
@@ -344,7 +344,7 @@ If there are any failures during self-hosted engine deployment, find the problem
 Make sure that the installation does not format the other disks, as bricks are created on top of those disks.
 ====
 
-.. If multipath configuration is not available on the newly installed host, blacklist the Gluster devices. The inventory file is already created in the first host as part of the step _Blacklist the devices used for Gluster bricks_.
+.. If multipath configuration is not available on the newly installed host, denylist the Gluster devices. The inventory file is already created in the first host as part of the step _Disable the devices used for Gluster bricks_.
```

Comment:
I think you should use 'disable' instead of 'denylist' here. The denylist was for a specific context -- preventing drivers from loading in the kernel. For Gluster, we are OK with 'disabling' devices.